tests/unit/store/flask.spec.js

```diff
@@ -299,35 +297,41 @@ describe("store/flask", () => {
           STATUS.MESSAGE.SHUTDOWN
         );
       });
-      test("When set_status_ping_interval_id is committed, the ID mutates", async () => {
+      test("When set_status_ping_interval_id is committed, Then ID mutates", async () => {
         const expected_id = 2993;
         store.commit("flask/set_status_ping_interval_id", expected_id);
-        expect(store.state.flask.status_ping_interval_id).toEqual(expected_id);
+        expect(store.state.flask.status_ping_interval_id).toStrictEqual(
+          expected_id
+        );
       });
 
-      test("When stop_status_pinging is committed, clearInterval is not called unnecessarily", async () => {
+      test("When stop_status_pinging is committed, Then clearInterval is not called unnecessarily", async () => {
         const spied_clear_interval = jest.spyOn(window, "clearInterval");
 
         store.commit("flask/stop_status_pinging");
-        expect(spied_clear_interval).not.toBeCalled();
+        expect(spied_clear_interval).not.toHaveBeenCalled();
       });
 
-      test("UUID setting for known ids", async () => {
+      test("When UUID setting for known ids, Then assert if the value in store is matching the UUID", async () => {
         const need_calibration_uuid = "009301eb-625c-4dc4-9e92-1a4d0762465f";
 
         store.commit("flask/set_status_uuid", need_calibration_uuid);
-        expect(store.state.flask.status_uuid).toEqual(need_calibration_uuid);
+        expect(store.state.flask.status_uuid).toStrictEqual(
+          need_calibration_uuid
+        );
       });
 
-      test("Simulation setting for known ids", async () => {
+      test("When simulation_mode is set to true, Then assert if the value in store is matching for 'true'", async () => {
         const need_simulation = true;
 
         store.commit("flask/set_simulation_status", need_simulation);
-        expect(store.state.flask.simulation_mode).toEqual(need_simulation);
+        expect(store.state.flask.simulation_mode).toStrictEqual(
+          need_simulation
+        );
       });
 
-      test("Simulation default settings is false", async () => {
-        expect(store.state.flask.simulation_mode).toEqual(false);
+      test("When simulation_mode is not set, Then assert simulation_mode default settings is false", async () => {
```

Comment:
The "When" section should describe something active that is happening--it should try and avoid just saying "not doing something".  I think you have something like this in other sections: `When Vuex is initialized to its default state, Then...`

Reply:
Corrected